.github/workflows/build_publish_builder_image.yaml

```diff
@@ -0,0 +1,73 @@
+name: Build and Push Builder Image
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
the PR description says:
> On pushes to main branch, it will ensure new images are released for all the go and OS matrix arch.

how are you ensuring it?

Reply:
I'm sorry but I didn't get the question. It specifies that on pushes to main, it will trigger for the matrix.

Reply:
just ignore it, I got it now 😄